external_executors/local_cromwell/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:xenial
```

Comment:
Please stick with the same docker base image for ubuntu that is currently being used (zesty).

Reply:
No problem. One concern is that zesty of officially EOL in a couple weeks. Do you know if we're planning on updating?